packages/venia-ui/lib/components/CheckoutPage/ShippingMethod/updateModal.js

```diff
@@ -27,36 +28,46 @@ const UpdateModal = props => {
     return (
         <Modal>
             <aside className={rootClass}>
-                <Form className={classes.contents} onSubmit={handleSubmit}>
+                <Form
+                    className={classes.contents}
+                    getApi={setFormApi}
+                    initialValues={formInitialValues}
+                    onSubmit={handleSubmit}
+                >
                     <div className={classes.header}>
                         <span className={classes.headerText}>
                             {'Edit Shipping Method'}
                         </span>
-                        <UpdateModalCancelIconButton
-                            className={classes.headerButton}
-                            onClick={handleCancel}
-                        />
+                        {!isLoading && (
```

Comment:
Better to move this to a variable outside the return and use that variable here. Also instead of using `&&` operator, better to use `if else` to maintain return value consistency.

![image](https://user-images.githubusercontent.com/35203638/81234595-ae5b5200-8fbe-11ea-9ca2-3a456ee3b8ea.png)

Reply:
Resolved by https://github.com/magento/pwa-studio/pull/2382/commits/ae1d682abbcd6dee3c0b7f071a3d32a7f78eafd1.